taskcluster/tc-build-utils.sh

```diff
@@ -337,6 +410,116 @@ do_nuget_build()
   nuget pack nupkg/deepspeech.nuspec
 }
 
+do_nuget_repackage()
+{
+  package_option=$1
+  PROJECT_NAME=$2
+  
+  if [ -z "${PROJECT_NAME}" ]; then
+    exit "Please call with a valid PROJECT_NAME"
+    exit 1
+  fi;
+
+  
+  # Forming nuget name
+  nuget="${PROJECT_NAME}.${DS_VERSION}.nupkg"
+
+  # Getting Nuget artifact URL
+  nuget_pkg_url=$(get_dep_nuget_pkg_url "${nuget}")
+    
+  cd ${DS_DSDIR}/native_client/dotnet
+  ${WGET} -O - "${nuget_pkg_url}" 
+  
+  cd ${DS_DSDIR}/native_client/dotnet/nupkg
+  gunzip > "${DS_DSDIR}/native_client/dotnet/${nuget}"
+  
+  local all_deps="$(curl -s https://community-tc.services.mozilla.com/api/queue/v1/task/${TASK_ID} | python -c 'import json; import sys; print(" ".join(json.loads(sys.stdin.read())["dependencies"]));')"
+  
+  for dep in ${all_deps}; do
+    local has_artifact=$(curl -s https://community-tc.services.mozilla.com/api/queue/v1/task/${dep}/artifacts | python -c 'import json; import sys; has_artifact = True in [ e["name"].find("'libdeepspeech.zip'") > 0 for e in json.loads(sys.stdin.read())["artifacts"] ]; print(has_artifact)')
+
+    cd ${DS_DSDIR}/native_client/dotnet
+    ${WGET} -O - "https://community-tc.services.mozilla.com/api/queue/v1/task/${dep}/artifacts/public/libdeepspeech.zip"
+
+    if [ "${has_artifact}" = "True" ]; then
+      if [ "${package_option}" = "--cuda" ]; then
```

Comment:
i dont think this switch approach is good: hardcodes too much, you should not have different repackage flow for cpu/gpu/tflite